sstables/uncompress.cc

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <cstdint>
+#include <seastar/core/fstream.hh>
+
+#include "seastar/core/future.hh"
+#include "seastar/core/temporary_buffer.hh"
```

Comment:
Use angle brackets for seastar includes; they are "system headers" from ScyllaDB's perspective.